src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -13,199 +13,385 @@ public enum SyntaxKind : ushort
         List = GreenNode.ListKind,
 
         // punctuation
+        /// <summary>Represents <c>~</c> token.</summary>
         TildeToken = 8193,
+        /// <summary>Represents <c>!</c> token.</summary>
         ExclamationToken = 8194,
+        /// <summary>Represents <c>$</c> token. It's a debugger special punctuation and not related to string interpolation.</summary>
```

Comment:
```suggestion
        /// <summary>Represents <c>$</c> token.</summary>
        /// <remarks>This is a debugger special punctuation and not related to string interpolation.</remarks>
```

Reply:
Wow! didn't know that remarks allowed for enum members. I only have seen them for classes, structs, and methods before. Great to know 🎉

Reply:
While allowed, I don't think the remark would be visible on docs.MS. For example, `LanguageVersion.CSharp3` has a remark:

https://github.com/dotnet/roslyn/blob/fd0e8520a8c01977e21772bd825526cb41e88a41/src/Compilers/CSharp/Portable/LanguageVersion.cs#L26-L32

[But it's not visible on docs.MS.](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.languageversion?view=roslyn-dotnet#Microsoft_CodeAnalysis_CSharp_LanguageVersion_CSharp3)

Reply:
@svick Hmm.. Is that a problem only with docFX? or any documentation generator will do the same?
I think that's worth adding a compiler warning for that :smile:
@sharwell What do you think? Should I revert and open an issue to propose a warning for this?